integration_test/not_connected_test.go

```diff
@@ -20,10 +22,20 @@ func (s *notConnectedSuite) SuiteName() string {
 
 func init() {
 	itest.AddTrafficManagerSuite("", func(h itest.NamespacePair) itest.TestingSuite {
-		return &notConnectedSuite{Suite: itest.Suite{Harness: h}, NamespacePair: h}
+		s := &notConnectedSuite{Suite: itest.Suite{Harness: h}, NamespacePair: h}
+		suffix := itest.GetGlobalHarness(h.HarnessContext()).Suffix()
+		s.serviceName = "echo"
+		s.appSpace, _ = itest.AppAndMgrNSName(suffix)
+		return s
 	})
 }
 
+func (s *notConnectedSuite) SetupSuite() {
+	s.Suite.SetupSuite()
+	ctx := s.Context()
+	itest.ApplyEchoService(ctx, s.serviceName, s.appSpace, 80)
```

Comment:
This will slow down all other tests in the `notConnectedSuite` when running them as singleton tests. So please don't do this.